### ISSUE TITLE
Wordpress installation failed

### ISSUE THREAD

**Issue by renesigi** [stephdl/nethserver-wordpress#5]
Hi Stephan

Just wanted to install wordpress:

yum install nethserver-wordpress --enablerepo=stephdl

and get the following error messages:
: 
--> Finished Dependency Resolution
Error: Package: php-pdo-5.4.16-45.el7.x86_64 (ce-base)
           Requires: php-common(x86-64) = 5.4.16-45.el7
           Installed: php-common-5.4.16-46.el7.x86_64 (@base)
               php-common(x86-64) = 5.4.16-46.el7
           Available: php-common-5.4.16-45.el7.x86_64 (ce-base)
               php-common(x86-64) = 5.4.16-45.el7
Error: Package: php-mbstring-5.4.16-45.el7.x86_64 (ce-base)
           Requires: php-common(x86-64) = 5.4.16-45.el7
           Installed: php-common-5.4.16-46.el7.x86_64 (@base)
               php-common(x86-64) = 5.4.16-46.el7
           Available: php-common-5.4.16-45.el7.x86_64 (ce-base)
               php-common(x86-64) = 5.4.16-45.el7
Error: Package: php-gd-5.4.16-45.el7.x86_64 (ce-base)
           Requires: php-common(x86-64) = 5.4.16-45.el7
           Installed: php-common-5.4.16-46.el7.x86_64 (@base)
               php-common(x86-64) = 5.4.16-46.el7
           Available: php-common-5.4.16-45.el7.x86_64 (ce-base)
               php-common(x86-64) = 5.4.16-45.el7
Error: Package: php-process-5.4.16-45.el7.x86_64 (ce-base)
           Requires: php-common(x86-64) = 5.4.16-45.el7
           Installed: php-common-5.4.16-46.el7.x86_64 (@base)
               php-common(x86-64) = 5.4.16-46.el7
           Available: php-common-5.4.16-45.el7.x86_64 (ce-base)
               php-common(x86-64) = 5.4.16-45.el7
 You could try using --skip-broken to work around the problem
 You could try running: rpm -Va --nofiles --nodigest

I have the newest version of nethserver installed config show:
[SITS-SRV02-Config-Show.txt](https://github.com/stephdl/nethserver-wordpress/files/2683791/SITS-SRV02-Config-Show.txt)

Can you please help me to solve this problem.

Thanks in advance -- René

**Comment by stephdl** [comment]
This is not related to nethserver-wordpress, I suggest to you to open a thread on community.nethserver.org
```

Error: Package: php-gd-5.4.16-45.el7.x86_64 (ce-base)
Requires: php-common(x86-64) = 5.4.16-45.el7
Installed: php-common-5.4.16-46.el7.x86_64 (@base)
php-common(x86-64) = 5.4.16-46.el7
Available: php-common-5.4.16-45.el7.x86_64 (ce-base)
php-common(x86-64) = 5.4.16-45.el7
```

In a first shoot i would think you could clean  the yum rempository and start it again

`yum clean all --enablerepo=*`

then verifiy which php-common you have installed

`yum info php-common`

**Comment by renesigi** [comment]
Hi Stephane

Thanks for fast answering. Finally, I was successful to install nethserver-wordpress, BUT only, because I have manually removed the php release 5.4.16-46-el7 and replaced it with 5.4.16-45-el7 using yum. But unfortunately this has broken my nethserver installation. So I do not have any access anymore to the server manager using https://server_name:980. Do you have any idea or links on how to repair the installation?

I've tried already nethserver-install, but that tells me:

NethServer is already installed!

You can access the Web interface at:

    https://sits.renesigi.ch:980 (or https://192.168.0.102:980)
    Login: root
    Password: <your_root_password>
Installation log can be found here: /var/log/nethserver-install.log

I can still login using SSH but as I said NOT  https://sits.renesigi.ch:980.

I know, has nothing to do with nethserver-wordpress, but some hints would would be anyway nice.

Kind Regards René
